### PR TITLE
(Fix): Include clean-up task in openebs-install jobs

### PR DESCRIPTION
**Pull request by amitbhatt818**
- This PR is for including openebs clean-up tasks in the  litmus job
Signed-off-by: Amit Bhatt <amitbhatt818@gmail.com>

### CODE REVIEW

File: utils/openebs-cleanup.yml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-- hosts: localhost
```
Comment (harshshekhar15) [inline utils/openebs-cleanup.yml]: Why is this change required?

Comment (amitbhatt818) [reply utils/openebs-cleanup.yml]: This change is required because when we include file at that time host is not required
```TASK [include_tasks] ***********************************************************
fatal: [127.0.0.1]: FAILED! => {"reason": "conflicting action statements: hosts, tasks\n\nThe error appears to be in '/utils/openebs-label-cleanup.yaml': line 1, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- hosts: localhost\n  ^ here\n"}
```
